CMakeLists.txt

```diff
@@ -410,13 +405,64 @@ else()
     set(DFHACK_TINYXML "dfhack-tinyxml")
 endif()
 
+if(BUILD_TESTING)
+    message("BUILD TESTS: Core, Scripts")
+    set(BUILD_SCRIPT_TESTS ON FORCE)
+    set(BUILD_CORE_TESTS ON FORCE)
+endif()
+
+find_package(ZLIB REQUIRED)
+
+if(BUILD_CORE_TESTS)
+    include_directories(depends/googletest/googletest/include)
+endif()
```

Comment:
The tinyxml block is another dependency. It may not be obvious, but the entire tinyxml block was formerly `include_directories(depends/tinyxml)`. So I think the BUILD_TESTING stuff should go outside, to avoid interrupting the list of dependencies here.

Reply:
You're gonna have to be specific, cause I put the stuff there to get variables setup before the depends subdir

Reply:
I've taken a best guess at what you meant